File: school/summer2013/openmp/fortran/32_task2.f90
program task2
    integer omp_get_thread_num

!$omp parallel num_threads(32)
    !$omp single
    print *, 'A tid=', omp_get_thread_num()
    print *, 'B tid=', omp_get_thread_num()
    print *, 'C tid=', omp_get_thread_num()
    print *, 'D tid=', omp_get_thread_num()
    !$omp end single
!$omp end parallel
end

File: summer2013/example_openmp/c/32_task2.c
#include <stdio.h>
#include <omp.h>

int main()
{
#pragma omp parallel num_threads(32)
{
	#pragma omp single
	{
		printf("A tid=%d\n", omp_get_thread_num() );
		printf("B tid=%d\n", omp_get_thread_num() );
		printf("C tid=%d\n", omp_get_thread_num() );
		printf("D tid=%d\n", omp_get_thread_num() );
	}
}
}
